### ISSUE TITLE
[require-yields-description] support bare tag with description in next line

### ISSUE THREAD

**Issue by danielrentz** [gajus/eslint-plugin-jsdoc#1528]
Thanks for providing the new rule "require-yields-description" so quickly. :)

However, in our code base it triggers where it shouldn't, I think.

## Expected behavior

"require-yields-description" should accept a bare `@yields` tag with its description starting in the next line. This works as expected for the `@returns` tag. We use this in TS code with suppressed type annotations for all tags (except `@throws`).

## Actual behavior

The rule fails on these tags with "@yields should have a description".

## ESLint Config

```js
rules: {
  "jsdoc/require-returns-description": "error",
  "jsdoc/require-yields-description": "error",
},
```

## ESLint sample

```ts
/**
 * @yields
 *  The results.
 */
export function *test1(): IterableIterator<string> { yield "hello"; }
```

All the following examples pass as expected.

```ts
/**
 * @yields The results.
 */
export function *test2(): IterableIterator<string> { yield "hello"; }

/**
 * @returns
 *  The result.
 */
export function test3(): string { return "hello"; }

/**
 * @returns The result.
 */
export function test4(): string { return "hello"; }
```

## Environment

- Node version: 22.14
- ESLint version 9.35
- `eslint-plugin-jsdoc` 59.0.0
